### PR TITLE
tests: Start osbuild-composer.socket 

### PR DESCRIPTION
**Pull request by atodorov**
otherwise API socket isn't automatically started

Related: rhbz#1916684

Should address the failures seen in https://github.com/cockpit-project/bots/pull/1578

### CODE REVIEW

**Comment by atodorov** [comment]
`composer-cli compose start test-http-server qcow2` fails. Likely with a depsolve problem.

**Review by bcl — APPROVED** [review]
Looks ok to me

File: tests/cli/lib/lib.sh [hunk 1]
```diff
@@ -108,7 +108,6 @@ composer_start() {
         # socket stop/start seems to be necessary for a proper service restart
         # after a previous direct manual run for it to work properly
         systemctl start $BACKEND.socket
-        systemctl start $BACKEND
```
Comment (martinpitt) [inline tests/cli/lib/lib.sh:111]: It feels a bit weird that the commit message  says "start osbuild-composer.socket" for this removal?

**Review by martinpitt — CHANGES_REQUESTED** [review]
`subscription-manager attach --auto` now [fails in the tests](https://logs.cockpit-project.org/logs/pull-1109-20210126-141717-4eb8d845-rhel-8-4-osbuild-composer/log.html)

**Review by martinpitt — APPROVED** [review]
Whee, thank you! :+1:

**Pull request merged**